bladex/profilebase.py

```diff
@@ -285,13 +287,18 @@ def max_camber(self, interpolate=False, n_interpolated_points=500):
         :return: maximum camber
         :rtype: float
         """
-        self.get_camber_line(interpolate=interpolate,
+        self.compute_camber_line(interpolate=interpolate,
                              n_interpolated_points=n_interpolated_points)
 
-        self.get_chord_line(lin_spaced=interpolate, 
+        self.compute_chord_line(lin_spaced=interpolate, 
                             num=n_interpolated_points)
 
-        camber = self.chord_line[1] + self.camber_line[1]
+        n_points = self.camber_line[0].size
+        camber = np.zeros(n_points)
+        for i in range(n_points):
+            camber[i] = np.linalg.norm(self.chord_line[:,i]
```

Comment:
this way the camber is always positive, but there are foils with negative camber.